bot/cogs/snekbox.py

```diff
@@ -152,7 +157,7 @@ def get_status_emoji(results: dict) -> str:
         lines = output.count("\n")
```

Comment:
Because we are now not stripping leading `'\n'` anymore, we will have to find a different way to count lines, otherwise we can have this

![image](https://user-images.githubusercontent.com/19921765/74163630-3d253080-4c55-11ea-8440-5959bfdda4b4.png)

I believe the leading `'\n'` should still be stripped, the leading `' '` *can* be left alone since it does mess up with output of `dis` and `pandas.DataFrame` if we also strip the `' '`

Reply:
The point was to leave all the output lines without doing any stripping logic to avoid any misleading result, since we have the paste service anyway

Reply:
I see, that is a good point, in this case please disregard this comment.